### PR TITLE
Add tests reports JUnit and coverage

### PR DESCRIPTION
**Pull request by yvon-dblg**
* Add dockerfile to test locally whith only docker installed.
* Remove travis build status from the README.
* Update gitignore accordingly
* Fix go.* files mode

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -54,6 +54,8 @@ pipeline {
             post {
                 always {
                     sh 'docker stop mongo4hydrotest${RUN_ID} && docker network rm hydrotest${RUN_ID}'
+                    junit 'test-report.xml'
+                    archiveArtifacts artifacts: 'coverage.html', allowEmptyArchive: true
```
Comment (ydrisR) [inline Jenkinsfile:58]: Missing `cobertura coberturaReportFile: 'coverage.xml'` to expose coverage file

File: Dockerfile.test [hunk 2]
```diff
@@ -0,0 +1,38 @@
+# Usage:
```
Comment (ydrisR) [inline Dockerfile.test:1]: Why a new docker file and not reusing our standard image "docker.ci.diabeloop.eu/go-build:1.15" ?

**Review by ydrisR — CHANGES_REQUESTED** [review]

File: Dockerfile.test (hunk 2 above)
Comment (yvon-dblg) [reply Dockerfile.test:1]: It's just for local testing, when you don't want to specifically start a mongodb for runnings theses tests.
It's not used in the Jenkins pipeline.
I found this docker script useful, so I propose it.
May not be included if you prefer.

**Review by ydrisR — APPROVED** [review]
OK

**Pull request merged**